html/components/SimpleHtml.js

```diff
@@ -25,6 +25,24 @@ import getEmptyObjectKeys from '../services/getEmptyObjectKeys';
 import isValidVideoFormat from '../services/isValidVideoFormat';
 import Image from './Image';
 
+const UNSPLASH_IMAGE_CONTAINER_ID = 'unsplash-image-container';
+
+function alterUnsplashImagesNode(node) {
+  const styleAttrib = _.get(node, 'attribs.style', '').trim();
+
+  if (node.name === 'div' && node.attribs.id === UNSPLASH_IMAGE_CONTAINER_ID) {
```

Comment:
Since `alterUnsplashImagesNode` is only called when `node.attribs.id === UNSPLASH_IMAGE_CONTAINER_ID`, isn't this if statement redundant? Maybe just check for `node.name === 'div'`?

Leaving to your discretion, maybe I missed something.

Reply:
You're right